### PR TITLE
release(v0.32.0-beta.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.32.0-beta.0 release.

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -6,19 +6,27 @@ github_repo = "siderolabs/omni"
 match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 pre_release = false
```
Comment (frezbo) [inline hack/release.toml:8]: i guess should be `true`?

**Review by frezbo — APPROVED** [review]

File: hack/release.toml (hunk 1 above)
Comment (frezbo) [reply hack/release.toml:8]: not a big deal

Comment (Unix4ever) [reply hack/release.toml]: fixed anyways :)

**Pull request merged**